### ISSUE TITLE
force_unwrapping rule catches IBOutlet Collections in swiftlint 0.9.1

### ISSUE THREAD

**Issue by tapi** [realm/SwiftLint#546]
IBOutlets are always force unwrapped and generally the force_unwrap rule ignores them.

However it appears as though with swiftlint 0.9.1 it is catching IBOutlet collections. ie.
```
@IBOutlet weak var constraint: NSLayoutConstraint!
```
is ignored by the linter but 
```
@IBOutlet var constraints: [NSLayoutConstraint]!
```
is not ignored

**Comment by brandenr** [comment]
I would think the IBOutelts should be caught by the force unwrap rule. They actually can be nil at times and cause a crash

**Comment by ikesyo** [comment]
They are not forced unwrapping, but are defining properties of implicitly unwrapped optional. Those two are not the same. So `force_unwrapping` rule is catching IUO would be the real issue.

**Comment by norio-nomura** [comment]
Thanks for filing issue.
This behavior has been changed by #537.
Maybe we need another approach for `force_unwrapping`.🤔

**Comment by norio-nomura** [comment]
I opened PR #552 for fixing the issue #546, #547.
Feedback is welcome.
